### DISCUSSION TITLE
zooming in around the mouse pointer? is it possible to do it?

### DISCUSSION THREAD

**Discussion by yodavid1984** [xournalpp/xournalpp#4181] [Q&A]
usually, once i zoom in (which i do a lot, since i'm using and handwriting on large files), i always have to select the hand tool right after and drag the view to the area i want, because it never focus on the region i want it to focus. it would be so much better for me if there was this option of zooming in but keeping the area around the mouse pointer in the center

**Comment by vladQll** [comment]
well I zoom in or out in the region I want simply by doing the following 1) put the cursor where in the specific region you want to zoom in or out 2) press ctrl and at the same time slide the mouse wheel forward if you want to zoom in or to the opposite side if you want to zoom out.

- the ideal is also that the mouse wheel also called central button in xournal is configured in mouse as hand, so that when you move away with the previous steps mentioned, you can also move with your hand pressing and dragging the mouse wheel.

here an example video

https://user-images.githubusercontent.com/28522011/183637647-a098b0ce-3f35-41f6-bc6b-7bd11d557643.mp4
